### PR TITLE
feat: Add conda module

### LINKED ISSUE #263: Add conda module
#### Describe the solution you'd like
The spaceship prompt shows the current Conda env -- this could be useful. A potential drawback is clutter with the python module, but I believe pyenv and conda env are not used together so this is irrelevant.

### PR DESCRIPTION
**Pull request by Scotsguy**
#### Description

This adds the `conda` module, mostly following [this implementation](https://github.com/denysdovhan/spaceship-prompt/blob/master/sections/conda.zsh)

#### Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

#### How Has This Been Tested?

- [ ] I have tested using **MacOS**
- [x] I have tested using **Linux**
- [ ] I have tested using **Windows**

#### Checklist:

- [x] I have updated the documentation accordingly.
- [x] I have updated the tests accordingly.

### CODE REVIEW

File: src/modules/mod.rs [hunk 1]
```diff
@@ -40,8 +41,16 @@ pub fn handle<'a>(module: &str, context: &'a Context) -> Option<Module<'a>> {
         "directory" => directory::module(context),
         "character" => character::module(context),
         "cmd_duration" => cmd_duration::module(context),
+        "conda" => conda::module(context),
         "dotnet" => dotnet::module(context),
         "env_var" => env_var::module(context),
+        "nodejs" => nodejs::module(context),
```
Comment (chipbuster) [inline src/modules/mod.rs]: Were these just all missing previously? :o

File: tests/testsuite/conda.rs [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+    let actual = String::from_utf8(output.stdout).unwrap();
+
+    assert_eq!(expected, actual);
+    Ok(())
+}
+
+#[test]
+fn env_set() -> io::Result<()> {
+    let output = common::render_module("conda")
+        .env_clear()
```
Comment (chipbuster) [inline tests/testsuite/conda.rs]: Just FYI, [Render module already does this for us](https://github.com/starship/starship/blob/master/tests/testsuite/common.rs#L32).

File: src/modules/mod.rs (hunk 1 above)
Comment (Scotsguy) [reply src/modules/mod.rs]: I honestly don't know how those got in there, must've been something during a rebase

File: tests/testsuite/conda.rs (hunk 2 above)
Comment (Scotsguy) [reply tests/testsuite/conda.rs]: I just copied these from the [aws testsuite](https://github.com/starship/starship/blob/master/tests/testsuite/aws.rs), basically

File: src/modules/mod.rs (hunk 1 above)
Comment (chipbuster) [reply src/modules/mod.rs]: I swear Git is just mocking me now...

Alright, if you don't mind, could you sort these into alphabetical order? Not entirely sure how they keep unsorting themselves, but we'd like to keep an order where we can (in accordance with the comment at the top of the `match`)

File: tests/testsuite/conda.rs (hunk 2 above)
Comment (chipbuster) [reply tests/testsuite/conda.rs]: Alright, looks like we'll have some cleanup work to do in the future. Thanks for letting me know.

**Pull request merged**